internal/utils/utils.go

```diff
@@ -64,3 +65,69 @@ func ExtractTextFromXML(file *zip.File) ([]string, error) {
 
 	return contents, nil
 }
+
+/*
+Wondering if this is even necessary but I can see how it can be useful
+as it also removes links, images, and code blocks.
+
+I'm not sure if this is the best way to remove markdown syntax.
+Inspired by https://github.com/mmatongo/site/blob/master/cmd/dnlm/helpers.go#L62-L87
+*/
+
+/* RemoveMarkdownSyntax removes markdown syntax from a string */
+func RemoveMarkdownSyntax(text string) string {
+	patterns := []string{
+		"(```[\\s\\S]*?```)",                      // Code blocks
+		"(`[^`\n]+`)",                             // Inline code
+		"!\\[([^\\]]*?)\\]\\(([^)]+)\\)",          // Images
+		"\\[([^\\]]+)\\]\\(([^)]+)\\)",            // Links
+		"(__|\\*\\*|_|\\*)(.+?)(__|\\*\\*|_|\\*)", // Bold and Italic
+		"~~(.+?)~~",                               // Strikethrough
+		"^#{1,6}\\s(.*)$",                         // Headers
+		"^>\\s(.*)$",                              // Blockquotes
+		"^-{3,}$",                                 // Horizontal rules
+		"^\\s*[\\*\\-+]\\s+(.+)$",                 // Unordered lists
+		"^\\s*\\d+\\.\\s+(.+)$",                   // Ordered lists
+	}
+
+	for _, pattern := range patterns {
+		re := regexp.MustCompile("(?m)" + pattern)
+		switch {
+		case strings.HasPrefix(pattern, "(```"):
+			text = re.ReplaceAllString(text, "$1")
+		case strings.HasPrefix(pattern, "(`"):
+			text = re.ReplaceAllString(text, "$1")
+		case strings.HasPrefix(pattern, "!\\["):
+			text = re.ReplaceAllString(text, "$1 ($2)")
+		case strings.HasPrefix(pattern, "\\["):
```

Comment:
```suggestion
		case strings.HasPrefix(pattern, `\[`):
```

Here (and for most other  `case` statement in the `switch`, the string literal notation would improve the readability

Reply:
I'll make a chore to clean this up